account_financial_report_qweb/wizard/general_ledger_wizard.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <!-- GENERAL LEDGER -->
+        <record id="ledger_general_wizard" model="ir.ui.view">
+            <field name="name">General Ledger</field>
+            <field name="model">ledger.report.wizard</field>
+            <field name="arch" type="xml">
+                <form>
+                    <group name="main_info">
+                        <field name="company_id"/>
+                    </group>
+                    <group name="date_currency_filter">
+                        <group name="date_ranger">
+                            <field name="date_range_id"/>
+                            <field name="date_from"/>
+                            <field name="date_to"/>
+                            <field name="fy_start_date" invisible="1"/>
+                        </group>
+                        <group name="extra_info">
+                            <field name="amount_currency"/>
+                            <field name="centralize"/>
+                        </group>
+                    </group>
+                    <group name="other_filters">
+                        <group name="moves">
+                            <field name="target_move" widget="radio"/>
+                        </group>
+                    </group>
+                    <label for="account_ids"/>
+                    <field name="account_ids" nolabel="1"/>
+                    <footer>
+                        <button name="button_view" string="View" type="object" default_focus="1" class="oe_highlight"/>
+                        or
+                        <button name="check_report_xlsx" string="Export XLSX" type="object"/>
```

Comment:
check_report_xlsx() is defined under ledger.report.wizard.line, not ledger.report.wizard